pkgs/tools/misc/osm2pgsql/default.nix

```diff
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "OpenStreetMap data to PostgreSQL converter";
     homepage = "https://osm2pgsql.org";
-    license = licenses.gpl2;
+    license = licenses.gpl2Plus;
```

Comment:
:+1: In accordance with [the license statement of upstream's README](https://github.com/openstreetmap/osm2pgsql/tree/52b68715ed06d84a381d638df208d596173482e5#license).